angular/projects/spark-angular/src/lib/components/inputs/sprk-checkbox-item/sprk-checkbox-item.component.ts

```diff
@@ -60,11 +60,21 @@ export class SprkCheckboxItemComponent implements OnInit {
   @Input()
   idString: string;
 
+  /**
+   * Determines whether the `sprk-b-InputContainer__visibility-toggle`
+   * class is applied.
```

Comment:
Lets expand on this comment a little more so that they know what its for vs. what specific class name it is since the class may not mean anything to the user. Maybe something like:

```
 /**
   * If `true`, the checkbox item will receive styling to make it a visiability toggle. Use this for "Show Password" checkboxes.